matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/sender/SenderInfo.kt

```diff
@@ -23,15 +23,15 @@ data class SenderInfo(
         /**
          * Consider using [disambiguatedDisplayName]
          */
-        val displayName: String?,
+        var displayName: String?,
         val isUniqueDisplayName: Boolean,
-        val avatarUrl: String?
+        var avatarUrl: String?
```

Comment:
I do not like this change. `data class` should be immutable. It will avoid using `!!` below, which is also a very bad practice.

Reply:
Done. Thank you.